emission/tests/storageTests/TestTimeSeries.py

```diff
@@ -81,6 +81,76 @@ def testExtraQueries(self):
         with self.assertRaises(AttributeError):
             list(ts.find_entries(time_query=tq, extra_query_list=[ignored_phones]))
 
+    def testFindEntriesCount(self):
+        '''
+        Test: Specific keys with other parameters not passed values.
+        Input: For each dataset: ["background/location", "background/filtered_location", "analysis/confirmed_trip"]
+            - Testing this with sample dataset: "shankari_2015-aug-21", "shankari_2015-aug-27"
+        Output: Aug_21: ([738, 508], [0]), Aug_27: ([555, 327], [0])
+            - Actual output just returns a single number for count of entries.
+            - Validated using grep count of occurrences for keys: 1) "background/location"     2) "background/filtered_location"
+                - $ grep -c <key> <dataset>.json
+        
+        For Aggregate Timeseries test case:
+        - The expected output would be summed-up values for the respective keys from the individual users testing outputs mentioned above.
+        - Output: ([1293, 835], [0])
+            - For each of the 3 input keys from key_list1: 
+                - 1293 = 738 (UUID1) + 555 (UUID2)
+                - 835 = 508 (UUID1) + 327 (UUID2)
+                - 0 = 0 (UUID1) + 0 (UUID2)
+
+        '''
+
+        ts1_aug_21 = esta.TimeSeries.get_time_series(self.testUUID1)
+        ts2_aug_27 = esta.TimeSeries.get_time_series(self.testUUID)
+
+        # Test case: Combination of original and analysis timeseries DB keys for Aug-21 dataset
+        key_list1=["background/location", "background/filtered_location", "analysis/confirmed_trip"]
+        count_ts1 = ts1_aug_21.find_entries_count(key_list=key_list1)
+        self.assertEqual(count_ts1, ([738, 508], [0]))
```

Comment:
why is this not `[0,0]` to match the `2x1` array from the timeseries for the two keys?

Reply:
The array size depends on the number of keys pertaining to either of the two timeseries databases.
`background keys` are two in number, hence two array elements.
Similarly `analysis keys` are one in total, hence just a single element.
If two analysis keys were present, then the returned array list would have two entries for each analysis key.

Reply:
https://github.com/e-mission/e-mission-server/pull/935#discussion_r1319156496